supervisor_client.py

```diff
@@ -0,0 +1,28 @@
+import http.client
+import socket
+import xmlrpc.client
+
+# This is a Python 3 implementation of using the XML-RPC interface
```

Comment:
This comment is not very helpful. If you want, write a doscring, explaining the purpose of the module, not the implementation details.